javascript/api/auth/emailCheck.js

```diff
@@ -0,0 +1,21 @@
+const emailDoubleCheck = async () => {
+  const inputEmail = document.querySelector('.email-input')
+  const errorEmailMessage = document.querySelector('.error-email-message')
+  const emailCheck = {
+    email: inputEmail.value
+  }
```

Comment:
혹시 따로 변수로 빼진 이유가 있을까요? 변수명이 의도에 맞는 이름이면 좋겠어요.

```suggestion
  const requestBody = {
    email: inputEmail.value
  }
```

Reply:
넵 감사합니다!! 변수로 선언 한 다음 호출을 해서 사용하면 좋겠다고 생각을 했었습니다. 강사님께서 말씀하신거에 대한 의도는 JSON.stringify() 안에 넣어서 할 수도 있는데 빼서 한 이유를 물어보시는게 맞을까요?